includes/wikia/models/DesignSystemGlobalFooterModel.class.php

```diff
@@ -516,13 +536,21 @@ private function getVerticalData() {
 	}
 
 	private function getLicenseData() {
-		$licenseText = WikiFactory::getVarValueByName( 'wgRightsText', $this->wikiId ) ?: $this->wg->RightsText;
+		if ( $this->product === self::FANDOM_PRODUCT ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)